make consul source fail

### Describe Operation

I make consul source by following the article `https://www.consul.io/docs/install/index.html`, but make consul  fail. when I run `make tools` in the directory`/Users/jianweilin/do_test/go/consul_source/src/github.com/hashicorp/consul`, show this errors

```
Fetching https://gopkg.in/matm/v1/gocov-html?go-get=1
Parsing meta tags from https://gopkg.in/matm/v1/gocov-html?go-get=1 (status code 200)
get "gopkg.in/matm/v1/gocov-html": found meta tag get.metaImport{Prefix:"gopkg.in/matm/v1/gocov-html", VCS:"git", RepoRoot:"https://gopkg.in/matm/v1/gocov-html"} at https://gopkg.in/matm/v1/gocov-html?go-get=1
gopkg.in/matm/v1/gocov-html (download)
github.com/matm/gocov-html (download)
package html: unrecognized import path "html" (import path does not begin with hostname)
make: *** [tools] Error 1
```


### relative environment variables:

#### 1. go environment

GO install in the directory:  $GOROOT

consul project in the directory:  $GOPATH/src/github.com/hashicorp

```
$ go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/jianweilin/do_test/go/consul_source"
GORACE=""
GOROOT="/usr/local/Cellar/go/1.9.1"
GOTOOLDIR="/usr/local/Cellar/go/1.9.1/pkg/tool/darwin_amd64"
GCCGO="gccgo"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/1s/1n5fnsgn54x5pq73bhqc3j4m0000gn/T/go-build354074163=/tmp/go-build -gno-record-gcc-switches -fno-common"
CXX="clang++"
CGO_ENABLED="1"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
```

#### 2. consul project path

```
/Users/jianweilin/do_test/go/consul_source/src/github.com/hashicorp/consul
```

--- a/README.md
+++ b/README.md
@@ -51,7 +51,7 @@ https://www.consul.io/docs
 ## Developing Consul
 
 If you wish to work on Consul itself, you'll first need [Go](https://golang.org)
-installed (version 1.9+ is _required_). Make sure you have Go properly installed,
+installed (version 1.10+ is _required_). Make sure you have Go properly installed,
 including setting up your [GOPATH](https://golang.org/doc/code.html#GOPATH).
 
 Next, clone this repository into `$GOPATH/src/github.com/hashicorp/consul` and

